iosHackthonApp/iosHackthonApp/Add Meal Screen/AddMealView.swift

```diff
@@ -7,11 +7,28 @@
 //
 
 import SwiftUI
+import shared
 
 struct AddMealView: View {
-    var body: some View {
-        Text(/*@START_MENU_TOKEN@*/"Hello, World!"/*@END_MENU_TOKEN@*/)
-    }
+	var body: some View {
+		//TODO: Pass the actual values when UI is complete
+		let sdk = PostMealApi()
```

Comment:
When you are updating this PR, could you pass the MealsSDK through to this view.
I know this is just a placeholder function just to check it works, but just so both of the screens are getting instantiated in the same way :-)

Reply:
Yup..done!